client_admin/pulp/client/admin/tasks.py

```diff
@@ -162,6 +175,33 @@ def cancel(self, **kwargs):
             else:
                 raise
 
+    def purge(self, **kwargs):
```

Comment:
If we don't want this behaviour in future, I can simply add this method to the AllTasksSection. It will be specific to all tasks then, not repo specific.